python/hyperon/exts/agents/agent_base.py

```diff
@@ -166,6 +170,41 @@ def __metta_call__(self, *args):
             return [E()]
         return st
 
+class BaseListeningAgent(AgentObject):
+    def __init__(self, path=None, atoms={}, include_paths=None, code=None):
+        super().__init__(path, atoms, include_paths, code)
+        self.messages = Queue()
+        self.running = True
```

Comment:
Should the agent be `running` after `__init__`, or should this happens after `start`?